### PR TITLE
Added LICENSE for PyDy

### LINKED ISSUE #15214: PyDy examples need to include the PyDy license
There is a new directory in `sympy/parsing/autolev/tests`:

```
│   ├── pydy-example-repo
│   │   ├── chaos_pendulum.al
│   │   ├── chaos_pendulum.py
│   │   ├── double_pendulum.al
│   │   ├── double_pendulum.py
│   │   ├── __init__.py
│   │   ├── mass_spring_damper.al
│   │   ├── mass_spring_damper.py
│   │   ├── non_min_pendulum.al
│   │   └── non_min_pendulum.py
```

These examples are copied from PyDy, but the PyDy license isn't included. I don't care personally, but we should probably follow protocol and include the PyDy license.

### PR DESCRIPTION
**Pull request by czgdp1807**
#### Brief description of what is fixed or changed
#15372 
Fixes #15214

#### Other comments
ping @certik @moorepants 

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by asmeurer** [comment]
CC @moorepants 

Who is the author of the examples? If the author(s) of the examples agree it can be included in SymPy under the SymPy license then we don't need to copy the PyDy licence.

**Comment by oscarbenjamin** [comment]
This looks good to me but I'd like to hear from someone from PyDy that this is ok

**Comment by czgdp1807** [comment]
ping @moorepants

**Comment by czgdp1807** [comment]
ping @moorepants

File: LICENSE [hunk 1]
```diff
@@ -95,3 +95,34 @@ CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
 LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
 OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
 DAMAGE.
+
+--------------------------------------------------------------------------------
+
+The files under the directory sympy/parsing/autolev/tests/pydy-example-repo
+are directly copied from PyDy project and are licensed as:
+
+Copyright (c) 2009-2017, PyDy Authors
```
Comment (moorepants) [inline LICENSE]:
```suggestion
Copyright (c) 2009-2019, PyDy Authors
```

**Comment by moorepants** [comment]
LGTM, with the date change.

**Comment by asmeurer** [comment]
Jason can you comment on https://github.com/sympy/sympy/pull/17927#issuecomment-555611451

**Comment by moorepants** [comment]
You'd have to ask all past contributors of PyDy.

**Comment by czgdp1807** [comment]
@moorepants Can we merge this?

**Comment by moorepants** [comment]
I think it is fine to merge as is.

**Comment by asmeurer** [comment]
It only matters for whoever wrote the code that has been included in SymPy.

**Comment by moorepants** [comment]
History:

https://github.com/pydy/pydy/commits/master/examples/mass_spring_damper

https://github.com/pydy/pydy/commits/master/examples/chaos_pendulum

Not quite sure where the nonmin pendulum problem is from. I remember seeing it somewhere but didn't find it right now.

**Comment by moorepants** [comment]
Here is the double pendulum: https://github.com/pydy/pydy/commits/master/examples/double_pendulum

**Comment by czgdp1807** [comment]
Well, I think including the license is better rather than asking PyDy developers. What do you say @moorepants ?

**Comment by czgdp1807** [comment]
ping @moorepants @asmeurer

**Review by certik — APPROVED** [review]
+1 to merge to be done with this. If PyDy developers give us permission to relicense the code, we can remove this text from `LICENSE`.

**Comment by czgdp1807** [comment]
Merging. If the license isn't needed then let me know I will make a follow up PR.

**Pull request merged**